xapian-letor/tests/api_letor.cc

```diff
@@ -229,6 +280,36 @@ DEFINE_TESTCASE(emptyfeaturelist, !backend)
     return true;
 }
 
+DEFINE_TESTCASE(bigfeaturelist, generated)
+{
+    vector<Xapian::Feature*> f;
+    f.push_back(new Xapian::TfFeature());
+    f.push_back(new Xapian::TfDoclenFeature());
+    f.push_back(new Xapian::IdfFeature());
+    f.push_back(new Xapian::CollTfCollLenFeature());
+    f.push_back(new Xapian::TfIdfDoclenFeature());
+    f.push_back(new Xapian::TfDoclenCollTfCollLenFeature());
+    f.push_back(new Xapian::TfFeature());
+    f.push_back(new Xapian::TfDoclenFeature());
```

Comment:
To avoid people having to check the feature implementations in future, it might be worth having comments saying how many values each feature contributes to the feature vector, which would explain where the number 25 comes from later in the testcase.

Reply:
fixed in a fixup commit.